.github/workflows/build.yaml

```diff
@@ -160,8 +160,21 @@ jobs:
           default: true
           components: clippy
           target: armv7-unknown-linux-gnueabihf
+      # Use zig as our C compiler for convenient cross-compilation. We run into rustls having a dependency on `ring`.
+      # This crate uses C and assembly code, and because of its build scripts, `cargo clippy` needs to be able to compile
+      # that code for our target.
+      - uses: goto-bus-stop/setup-zig@v2
```

Comment:
This should be pinned to a specific commit to be consistent with the other actions.